benchmark/buffers/buffer-tostring.js

```diff
@@ -3,10 +3,10 @@
 const common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
-  encoding: ['', 'utf8', 'ascii', 'latin1', 'binary', 'hex', 'UCS-2'],
-  args: [0, 1, 2, 3],
-  len: [0, 1, 64, 1024],
-  n: [1e7]
+  encoding: ['utf8', 'ascii', 'latin1', 'hex', 'UCS-2'],
+  args: [0, 1, 3],
+  len: [1, 64, 1024],
+  n: [1e6]
```

Comment:
The `''` encoding is identical to `'utf8'` and `'binary'` is identical to `'latin1'`.

The middle args should not have any actual influence on the performance profile in case no args and all args do not deviate significantly (which they never should due to just setting some defaults and the actual work is done afterwards). So we might actually remove all args cases and just use a single type to benchmark the actual important part: toString.

The zero byte buffer is redundant to one byte and even that should normally not be important at all.